Server/Qoality_of_Life_changer.Model/AuthModel/AuthData.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Quality_of_Life_changer.Model.Auth;
+﻿namespace Quality_of_Life_changer.Model.AuthModel;
```

Comment:
Why do you need `AuthModel`, not `Auth`?
you have namespace hierarchy like `Model.<something>`, do not repeat `model` word everywhere